### PR TITLE
Explaining IPv6 support and improving custom domain instructions

### PR DESCRIPTION
**Pull request by brittag**
Here's an effort at a few things:
- Explain in our prospective-customer materials (specifically on [What's Included](https://docs.cloud.gov/intro/pricing/whats-included/)) that cloud.gov is fine if you need IPv6 support.
- Move the cloud.gov team instructions out of the [user-facing custom domain page](https://docs.cloud.gov/apps/custom-domains/) to an ops-team page. They probably need some updating, but for now I just moved them.
- Restructure the [user-facing custom domain page](https://docs.cloud.gov/apps/custom-domains/) to guide people along the process better - that they need to talk to us for some parts of the process. Also include IPv6 mentions for customers who want to be sure that IPv6 is addressed and supported.

### CODE REVIEW

**Comment by wslack** [comment]
Nothing is meant to point to the operational doc, right? I don't see any links to it, wanted to verify nothing needs to point to it. This LGTM and I am 👍 to merge but wanted someone else's eyes on the team.

File: content/apps/custom-domains.md [hunk 1]
```diff
@@ -152,7 +94,8 @@ cloud.gov offers a managed service that allows you to forward a domain you contr
     Message: Provisioning in progress; CNAME domain "my.domain.gov" to "d3kajwa62y9xrp.cloudfront.net."
     ```
 
-1. Create/update your DNS configuration.
+1. Create/update your DNS configuration (including your IPv6 AAAA configuration if you need it).
```
Comment (dlapiduz) [inline content/apps/custom-domains.md]: Sorry for the confusion... ipv6 is not available in cloudfront just in ELBs so we have to use the "manual method" to make it work.

File: content/apps/custom-domains.md [hunk 2]
```diff
@@ -9,102 +9,44 @@ title: Custom Domains
 
 Here's how to put your app on your project's custom domain name. For context, [Cloud Foundry's Routes and Domains documentation](https://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html) explains the overall model and terminology that cloud.gov uses.
 
-The [Manual Method](#manual-method) is available to all users. We've also developed a [Managed Service Method](#managed-service-method), but this is not automatically available to users yet because it's in cloud.gov's GovCloud environment (if you need this, [let us know](/help/)).
+The [Manual Method](#manual-method) is available to all users. We've also developed a [Managed Service Method](#managed-service-method), but this is not automatically available to most users yet because it's in cloud.gov's GovCloud environment (if you need this, [let us know](/help/)).
+
+Both of these methods support user access to your application over IPv6 if your DNS supports IPv6.
```
Comment (wslack) [inline content/apps/custom-domains.md]: Given what @dlapiduz said, I think this should be updated as well?

**Comment by brittag** [comment]
@wslack Yeah, I figure the operational doc would just live in the ops section (without being explicitly linked from the user-facing docs, since it's not of use to users).

**Pull request merged**